source/jormungandr/jormungandr/interfaces/v1/Journeys.py

```diff
@@ -120,6 +120,9 @@ def wrapper(*args, **kwargs):
             if has_invalid_reponse_code(objects) or journeys_absent(objects):
                 return objects
 
+            # This is a temporary hack to resolve Error introduced during migration to Python3
```

Comment:
If this is a temporary hack, what will be the long term solution?

Reply:
As this is the very first error found on args parsing in Python3, I suppose we can wait for other errors to be detected and hence do more effort to find/make better solution :smile:

Reply:
`request.args` is a global variable from slack used to store arguments parsed previously which is buggy in python3, very likely due to the (extremely) old version of Flask that we are using...

Reply:
Maybe we can clarify in comment (TODO?) that this is supposed to be removed once flask is upgraded, then?
Or that we might also just wait for jormungandr to be replaced.